tests/containers/docker.pm

```diff
@@ -51,7 +54,12 @@ sub run {
     test_seccomp();
     allow_selected_insecure_registries(runtime => 'docker');
 
-    check_docker_firewall() if $self->firewall() eq 'firewalld';
+    if ($self->firewall() eq 'firewalld') {
+        # on publiccloud we need to install firewalld first
+        install_and_start_firewalld() if (is_publiccloud || is_jeos);
```

Comment:
@mloviska WDYT ?

Reply:
As we do not have a subroutine to determine between types of jeos images, I guess this way it is fine for now.